step/step.go

```diff
@@ -221,19 +222,18 @@ func (s XcodeTestConfigParser) ProcessConfig() (Config, error) {
 
 // InstallDeps ...
 func (s XcodeTestRunner) InstallDeps() error {
```

Comment:
InstallDeps isn't returning an error now. I suggest removing the error return value from the function signature.